sdk/core/azure-core/src/datetime.cpp

```diff
@@ -209,7 +209,8 @@ std::string DateTime::ToString(DateFormat format) const
           leftover);
 
       outCursor += 19;
-      if (fracSec != 0)
+      if ((fracSec != 0 && fractionFormat != TimeFractionFormat::Truncate)
+          || fractionFormat == TimeFractionFormat::AllDigits)
       {
         // Append fractional second, which is a 7-digit value with no trailing zeros
```

Comment:
This comment needs to be updated now because it is conditional.

Reply:
nit?

Reply:
yes :)